build/docker/builder.dockerfile

```diff
@@ -0,0 +1,35 @@
+# Dockerfile for building kmesh-build docker image.
+#
+# Usage:
+# docker build -f builder.dockerfile -t kmesh-build:latest .
+#
+
+# base image
+FROM openeuler/openeuler:23.09
+
+# Setup Go
+COPY --from=golang:1.22.7 /usr/local/go/ /usr/local/go/
```

Comment:
Do we need to hardcode the golang version? Can we write the golang version as latest?

Reply:
I think we can do this manually. To be determined